landing_page_backend/contacts/utils.py

```diff
@@ -8,20 +8,16 @@
 file_name = f'contacts-{date}.csv'
 one_week_ago = date - timedelta(days=7)
 
-with open(f'contacts/reports/{file_name}', 'w') as file:
-    writer = csv.writer(file)
-    writer.writerow(
-        ('name', 'email', 'number', 'message')
-    )
-
-
-def write_data_to_csv_file():
-    contacts = Contact.objects.filter(time_create__gte=one_week_ago).values_list('name', 'email', 'number', 'message')
-
-    for contact in contacts:
-        file = open(f'contacts/reports/{file_name}', 'a')
-        with file:
-            writer = csv.writer(file)
-            writer.writerow(
-                contact
-            )
+
+def export_contacts_to_csv_file():
+    contacts = Contact.objects.filter(create__gte=one_week_ago).values_list('name', 'email', 'number', 'message')
+
+    file = open(f'contacts/reports/{file_name}', 'w', newline='')
+
+    with file:
+        header = ['name', 'email', 'number', 'message']
+
+        write = csv.writer(file)
+        write.writerow(header)
+        for fields in contacts:
```

Comment:
That is a good example of a bad naming.
Field is usually a name or email. Reading this code I think field contains a name or email, while it is a tuple with all fields for a specific contact. That is worse than a code, where I cannot understand the purpose. That is misleading, like I go to the furniture shop and there is no furniture, but they sell cars. Please remember this example, it is very important for code readability.

Rename fields to contact_data.